keyboards/darkproject/kd83a_bfg_edition/info.json

```diff
@@ -0,0 +1,104 @@
+{
+    "keyboard_name": "KD83A_BFG_Edition",
+    "url": "",
+    "maintainer": "GSKY",
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
+                { "label": "ESC", "x": 0, "y": 0 },
+
+                { "label": "F1", "x": 1.25, "y": 0 },
+                { "label": "F2", "x": 2.25, "y": 0 },
+                { "label": "F3", "x": 3.25, "y": 0 },
+                { "label": "F4", "x": 4.25, "y": 0 },
+
+                { "label": "F5", "x": 5.5, "y": 0 },
+                { "label": "F6", "x": 6.5, "y": 0 },
+                { "label": "F7", "x": 7.5, "y": 0 },
+                { "label": "F8", "x": 8.5, "y": 0 },
+
+                { "label": "F9", "x": 9.75, "y": 0 },
+                { "label": "F10", "x": 10.75, "y": 0 },
+                { "label": "F11", "x": 11.75, "y": 0 },
+                { "label": "F12", "x": 12.75, "y": 0 },
+
+                { "label": "Printscreen", "x": 14, "y": 0 },
+                { "label": "Home", "x": 15.5, "y": 0 },               
```

Comment:
```suggestion
                { "label": "Home", "x": 15.5, "y": 0 },
```

Reply:
@fauxpark changes are made. Could you please review it.

Reply:
@fauxpark Could you please review the changes.